docs/user_guides/llm/vertexai/README.md

```diff
@@ -0,0 +1,102 @@
+    "Where was its 44th president born?",
+]
+
+for prompt in prompts:
+    print("User:", prompt)
+    print("Gemini:", get_chat_response(chat, prompt))
+    print("------")
+```
+
+```
+User: Hi, who are you?
+Gemini: I am Gemini, a multimodal AI model, developed by Google.
+------
+User: What can you tell me about the United States?
+Gemini: The United States of America, often referred to as the US or America, is a country primarily located in North America. It consists of 50 states, a federal district, five major unincorporated territories, 326 Indian reservations, and nine minor outlying islands. At 3.8 million square miles, the United States is the world's third or fourth-largest country by total area and is slightly smaller than the entire continent of Europe's 3.9 million square miles. With a population of over 332 million, it is the third most populous country in the world. The capital is Washington, D.C., and the most populous city is New York City.
+
+The United States is a federal republic and a representative democracy, in which the government is chosen by the people. The country is divided into three branches of government: the legislative branch, the executive branch, and the judicial branch. The legislative branch is composed of the Senate and the House of Representatives, which together form the United States Congress. The executive branch is led by the President, who is also the commander-in-chief of the armed forces. The judicial branch is composed of the Supreme Court and lower federal courts.
+
+The United States has a diverse economy, with a gross domestic product (GDP) of over $20 trillion. The country is a major exporter of agricultural products, machinery, and technology. The United States is also a major importer of oil and other natural resources.
+
+The United States is a member of the United Nations, NATO, the G7, the G20, and the Organisation for Economic Co-operation and Development (OECD). The country is a permanent member of the United Nations Security Council.
+
+The United States has a long and complex history. The first European settlers arrived in the early 17th century. The United States gained independence from Great Britain in 1776. The country has been involved in a number of wars, including the American Civil War, the Spanish-American War, and World War II.
+
+The United States is a diverse country, with people from all over the world. The country has a rich culture, with a wide variety of music, art, and literature. The United States is also a major center of technology and innovation.
+------
+User: Where was its 44th president born?
+Gemini: The 44th President of the United States, Barack Obama, was born in Honolulu, Hawaii on August 4, 1961.
+------
+```
+
+Now we can test Vertex AI models with GuardRails. The `config` directory includes a variation of the ABC Bot but calling the Gemini 1.0 Pro model from Vertex AI. The main difference is in the `config.yml` file. The relevant section is
```

Comment:
We should not duplicate the config in this folder. The notebook should copy the config when running and update the models part. The config folder itself should also be added to `.gitignore` so it doesn't get committed.

Reply:
Updated in commit [5319dbc5b89201e1e4d7046386183dbd7c3b4a12](https://github.com/NVIDIA/NeMo-Guardrails/pull/402/commits/5319dbc5b89201e1e4d7046386183dbd7c3b4a12).